lib/workos/organizations.rb

```diff
@@ -74,29 +74,37 @@ def get_organization(id:)
       # @param [Array<Hash>] domain_data List of domain hashes describing an orgnaization domain.
       # @option domain_data [String] domain The domain that belongs to the organization.
       # @option domain_data [String] state The state of the domain. "verified" or "pending"
-      # @param [Array<String>] domains List of domains that belong to the organization.
-      #   Deprecated: Use domain_data instead.
       # @param [String] name A unique, descriptive name for the organization
-      # @param [Boolean, nil] allow_profiles_outside_organization Whether Connections
-      #  within the Organization allow profiles that are outside of the Organization's configured User Email Domains.
       # @param [String] idempotency_key An idempotency key
+      # @param [Boolean, nil] allow_profiles_outside_organization Whether Connections
+      #   within the Organization allow profiles that are outside of the Organization's configured User Email Domains.
+      #   Deprecated: If you need to allow sign-ins from any email domain, contact suppport@workos.com.
+      # @param [Array<String>] domains List of domains that belong to the organization.
+      #   Deprecated: Use domain_data instead.
       def create_organization(
         domain_data: nil,
         domains: nil,
         name:,
         allow_profiles_outside_organization: nil,
         idempotency_key: nil
       )
-        warn_deprecation '`domains` is deprecated. Use `domain_data` instead.' if domains
+        body = { name: name }
+        body[:domain_data] = domain_data if domain_data
```

Comment:
Previously we were unconditionally passing `domain_data`, which meant it was always in the request body, even if no value was passed.